src/commands/releases.rs

```diff
@@ -698,10 +698,14 @@ fn execute_files_upload<'a>(
 }
 
 fn get_url_prefix_from_args<'a, 'b>(matches: &'b ArgMatches<'a>) -> &'b str {
-    matches
+    let mut rv = matches
         .value_of("url_prefix")
-        .unwrap_or("~")
-        .trim_end_matches('/')
+        .unwrap_or("~");
+    // remove a single slash from the end.  so ~/ becomes ~ and app:/// becomes app://
+    if rv.ends_with('/') {
+        rv = &rv[..rv.len() - 1];
```

Comment:
Can we at least something add like:
`info!("Removed trailing slash from url-prefix")`
to have more insight for the user?

Reply:
The trailing slash will always be removed though. We never documented `~` to be a valid prefix, we always told people to supply `~/`. So the removing is the default case. It's only thrown away because some code later adds it.

Reply:
I don't think this is the correct fix for https://github.com/getsentry/sentry-cli/issues/311

There shouldn't be any stripping.

My usecase requires that the prefix be `app:///` - I think OP requires it to be `app://`

Why should we have to add a slash to what we need only to have it removed?

Reply:
@josh-m-sharpe At least in the example it seems that the stack trace contains `app:///`, so this should be fine